.rubocop.yml

```diff
@@ -1,43 +1,27 @@
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
-
-# TODO: remove this file by either moving cops here or fixing code
 inherit_from:
   - .rubocop_todo.yml
 
+inherit_gem:
+  theforeman-rubocop:
+    - rules/base.yml
+    - rules/ruby_target.yml
+    - rules/style.yml
+    - rules/metrics.yml
+    - rules/performance.yml
+    - rules/rails.yml
+    - rules/minitest.yml
+    - rules/style_lenient.yml
+
 AllCops:
-  TargetRubyVersion: 2.5
-  TargetRailsVersion: 5.2
-  Exclude:
-    - 'db/schema.rb'
-    - 'node_modules/**/*'
-    - 'vendor/**/*'
+  NewCops: disable
 
 Rails:
   Enabled: true
```

Comment:
This should be covered by `rules/rails.yml`:
https://github.com/theforeman/theforeman-rubocop/blob/a4e936fe00d4b839dc027b939f7d364b14faecd2/rules/rails.yml#L7-L8